docs/examples/15_minutes_to_QCoDeS.ipynb

```diff
@@ -195,7 +209,14 @@
    "source": [
     "## Instrument\n",
     "\n",
-    "`Instrument` class in Qcodes is responsible for holding connections to hardware, creating a parameter, function, or method for each piece of functionality of the instrument. For more information on instrument class you can look at the [detailed description here](http://qcodes.github.io/Qcodes/user/intro.html#instrument)  or in the [api documentation](http://qcodes.github.io/Qcodes/api/instrument/index.html) regarding instruments. "
+    "`Instrument` class in Qcodes is responsible for holding connections to hardware, creating a parameter, function, or method for each piece of functionality of the instrument. For more information on instrument classwe refer to the [detailed description here](http://qcodes.github.io/Qcodes/user/intro.html#instrument)  or the corresponding [api documentation](http://qcodes.github.io/Qcodes/api/instrument/index.html). "
```

Comment:
please remove the notion of a "function", we are in a (long) process of deprecating them :)